### PR TITLE
Matched 23 functions in m_scene

### PR DESCRIPTION
**Pull request by emilybrooks**
I had to fix a lot of functions that use objectExchangeBank instead of objectExchangeBank->status, so there's a couple edits to m_actor.c and m_submenu.c

### CODE REVIEW

File: include/m_scene.h [hunk 1]
```diff
@@ -4,11 +4,92 @@
...
+void mSc_dmacopy_all_exchange_bank(ObjectExchangeBank* objectExchangeBank);
+void mSc_data_bank_ct(struct Game_Play* game_play, ObjectExchangeBank* objectExchangeBank);
+void mSc_decide_exchange_bank(ObjectExchangeBank* objectExchangeBank);
+// void func_800C6690_jp();
+void Scene_ct(struct Game_Play* game_play, void* arg1);
+// void func_800C6960_jp();
+void Scene_Proc_Ctrl_Actor_Ptr(struct Game_Play* /*game_play*/, MSceneUnkStruct* /*arg1*/);
+void Scene_Proc_Actor_Ptr(struct Game_Play* /*game_play*/, MSceneUnkStruct* /*arg1*/);
+void Scene_Proc_Object_Exchange_Bank_Ptr(struct Game_Play* /*game_play*/, MSceneUnkStruct* /*arg1*/);
+void Scene_Proc_Door_Data_Ptr(struct Game_Play* /*game_play*/, MSceneUnkStruct* /*arg1*/);
```
Comment (AngheloAlf) [inline include/m_scene.h:82]: Could you remove the comments?
```suggestion
void Scene_Proc_Ctrl_Actor_Ptr(struct Game_Play* game_play, MSceneUnkStruct* arg1);
void Scene_Proc_Actor_Ptr(struct Game_Play* game_play, MSceneUnkStruct* arg1);
void Scene_Proc_Object_Exchange_Bank_Ptr(struct Game_Play* game_play, MSceneUnkStruct* arg1);
void Scene_Proc_Door_Data_Ptr(struct Game_Play* game_play, MSceneUnkStruct* arg1);
```

File: include/m_common_data.h [hunk 2]
```diff
@@ -161,6 +161,19 @@ typedef struct FamicomEmuCommonData {
...
+    /* 0x0014 */ UNK_TYPE unk_14;
+    /* 0x0018 */ UNK_TYPE1 unk_18[0xF404];
+    /* 0xF41C */ u8 snowmanData[0xC];
+    /* 0xF428 */ u8 unk_F428[0x485];
+    /* 0xF8AD */ u8 snowmanYear;
+    /* 0xF8AE */ u8 snowmanMonth;
+    /* 0xF8AF */ u8 snowmanDay;
+    /* 0xF8B0 */ u8 snowmanHour;
+    /* 0xF8B1 */ u8 unk_F8B1[0x74F];
+} Save; // size = 0x10000
```
Comment (AngheloAlf) [inline include/m_common_data.h:175]: What is this struct?

Comment (Prakxo) [reply include/m_common_data.h:175]: It's the save structure, it's a substructure in common_data and I believe it's used and referenced in some files
I can investigate and give you a better reply later lol

Comment (emilybrooks) [reply include/m_common_data.h:175]: oh heck i think i made this a long time ago and didn't notice it was still in this pr, i'll remove it

**Review by AngheloAlf — APPROVED** [review]

**Comment by Prakxo** [comment]
I'm going to try to figure out the unnamed functions in m_scene here by looking at GC, I already found some, so don't merge yet!

**Review by Prakxo — APPROVED** [review]
I didn't checked the unmatched ones, but I noted some stuff

**Pull request merged**